x-pack/plugins/cloud_security_posture/public/pages/rules/rules_table_header.tsx

```diff
@@ -53,15 +53,16 @@ const SearchField = ({
   useDebounce(() => search(localValue), SEARCH_DEBOUNCE_MS, [localValue]);
 
   return (
-    <EuiFlexItem grow={true} style={{ alignItems: 'flex-end' }}>
+    <EuiFlexItem grow={true} style={{ alignItems: 'flex-end', maxWidth: 500 }}>
       <EuiFieldSearch
         isLoading={isSearching}
         placeholder={i18n.translate('xpack.csp.rules.rulesTable.searchPlaceholder', {
-          defaultMessage: 'Search',
+          defaultMessage: 'Search by rule name',
```

Comment:
technically, [placeholders](https://www.w3.org/WAI/tutorials/forms/instructions/#:~:text=%3E-,Placeholder%20text,when%20users%20start%20entering%20text.) should be an example of the input the user is expected to use. but it kinda looks bad to put a long sentence there: "Ensure ..." so i went with `by rule name`